content/introduction/use-cases/vc-your-app.md

```diff
@@ -22,9 +22,22 @@ Dolt ships with standard [RDBMS](../../concepts/dolt/rdbms/README.md) tools like
 
 In the past applications that needed these features required [slowly changing dimension](https://www.dolthub.com/blog/2021-09-17-database-version-control/) or [soft deletes](https://www.dolthub.com/blog/2022-11-03-soft-deletes/). These approaches are cumbersome and do not support merge. Dolt gives application the full development power of Git.
 
+# Dolt replaces
+
+## Soft Deletes
+
+A common technique to version your database is to use [soft deletes](https://www.dolthub.com/blog/2022-11-03-soft-deletes/). When your application would make an update or a delete, you application instead makes an insert and marks the old row invalid. Dolt obviates the need for this technique. You can keep your existing database schema and Dolt ensures every write is non-destructive. Queries against soft deleted rows become Dolt history queries against [system tables](../../reference/sql/version-control/dolt-system-tables.md). 
+
+## Slowly Changing Dimension
+
+A more advanced technique for versioning databases is [slowly changing dimension](https://en.wikipedia.org/wiki/Slowly_changing_dimension). Slowly Changing Dimension is similar to soft deletes. Additional database columns are added to tables to manage versioning. Dolt is slowly changing dimension on every table by default. Queries involving the slowly changing dimension become Dolt history queries against [system tables](../../reference/sql/version-control/dolt-system-tables.md). Moreover, complicated [merge](../../concepts/dolt/git/merge.md) processes can happen at the database layer. Merges must handled by custom code at the application layer with slowly changing dimension.
```

Comment:
This would be a good aaron blog. I still have trouble articulating the differences and tradeoffs of different schema versioning tactics

Reply:
I think Zach might attempt a slowly changing dimension blog firday